pkgs/top-level/all-packages.nix

```diff
@@ -5112,6 +5112,8 @@ in
 
   ltwheelconf = callPackage ../applications/misc/ltwheelconf { };
 
+  lunar-client = callPackage ../games/lunar-client {};
+
```

Comment:
Whoops, this needs to go in the lunar-client commit as well - together with the actual package code. Terribly sorry for missing this! :sweat:

Reply:
nah, I totally missed that too!